### DISCUSSION TITLE
Collecting Attacker Feeds

### DISCUSSION THREAD

**Discussion by AzizKpln** [telekom-security/tpotce#1546] [Q&A]
Hi, I'd like to collect all the feeds that's been detected by tpotce in 24 hours and block them in SIEM. I want to do this automatically for every 24 hours. Is there a way in this tool to do this? or do I have to use Crontab actions

**Comment by t3chn0m4g3** [comment]
[GreedyBear](https://github.com/intelowlproject/GreedyBear) is a good starting point.
